aws-rds-customdbengineversion/aws-rds-customdbengineversion.json

```diff
@@ -122,14 +134,23 @@
     "/properties/EngineVersion",
     "/properties/DatabaseInstallationFilesS3BucketName",
     "/properties/DatabaseInstallationFilesS3Prefix",
+    "/properties/ImageId",
     "/properties/KMSKeyId",
-    "/properties/Manifest"
+    "/properties/Manifest",
+    "/properties/SourceCustomDbEngineVersionIdentifier",
+    "/properties/UseAwsProvidedLatestImage"
   ],
   "handlers": {
     "create": {
       "permissions": [
+        "ec2:CopySnapshot",
+        "ec2:DeleteSnapshot",
+        "ec2:DescribeSnapshots",
         "kms:CreateGrant",
+        "kms:Decrypt,",
```

Comment:
Type of having ',' inside

Reply:
Thanks for catching this typo. Updating the PR